typings/resin-sdk.d.ts

```diff
@@ -418,10 +419,23 @@ declare namespace ResinSdk {
 			loginWithToken: (authToken: string) => Promise<void>;
 			logout: () => Promise<void>;
 			getToken: () => Promise<string>;
+			whoami: () => Promise<string>;
```

Comment:
In the docs this is `Promise<string | undefined>` (not that it would make huge difference).

Reply:
Should we also add
```
isLoggedIn: () => Promise<boolean>;
getUserId: () => Promise<number>;
getEmail: () => Promise<string>;
```
?

Reply:
Yes to both - done.